backend/src/Designer/Services/Implementation/ResourceRegistryService.cs

```diff
@@ -161,5 +159,26 @@ private StatusCodeResult GetPublishResponse(HttpResponseMessage response)
                 return new StatusCodeResult(400);
             }
         }
+
+        private string GetResourceRegistryBaseUrl(string env)
+        {
+            if (!_resourceRegistrySettings.TryGetValue(env, out ResourceRegistryEnvironmentSettings envSettings))
+            {
+                throw new ArgumentException($"Invalid environment. Missing environment config for {env}");
+            }
+
+            return envSettings.ResourceRegistryEnvBaseUrl;
+        }
+
+        private MaskinportenClientSettings GetMaskinportenIntegrationSettings(string env)
+        {
+            string maskinportenEnvironment = env == "prod" ? "prod" : "test";
```

Comment:
Er det noen appsettingsfiler som må oppdateres med disse verdiene?

Reply:
appsettings.json skal være oppdatert med maskinporten-settings for både test og prod. Slik at når kallet gjøres med "prod" satt som env-variabel vil den bruke prod-instillingene for maskinporten mens dersom det er et AT miljø vil den defaulte til test instillingene.